install/install.py

```diff
@@ -113,10 +113,12 @@ def finish():
 
 
 def setup_virtualenv():
-    Colorizer.info("[*] Seting up virtual environment named owtf...")
+    Colorizer.info("[*] Setting up virtual environment named owtf...")
+    # If /usr/local/bin/virtualenvwrapper.sh doesn't exist, create a symlink from /usr/bin/
+    run_command('sudo ln -s /usr/bin/virtualenvwrapper.sh /usr/local/bin/virtualenvwrapper.sh >/dev/null 2>&1;')
```

Comment:
@viyatb , ln -s will only create the symlink if it doesn't exist. If the file/symlink doesn't exist, it will quietly move on without making any change. Also, installation is only a one-time process and I don't think checking if the file exists or not is necessary.